x-pack/examples/embedded_lens_example/public/app.tsx

```diff
@@ -158,11 +158,14 @@ export const App = (props: {
                     <EuiButton
```

Comment:
Can we also pass in an `href` to make this render as an `<a>` element?

Reply:
@myasonik i thought about that as well, but i think EUI button Component doesn't allow passing href and onClick on the same component as far as i remember. 

ESLint: <EuiButton> accepts either `href` or `onClick`, not both.(@elastic/eui/href-or-on-click)

Reply:
This will also be problematic if you copy the link or use Right click mouse to Click Open in new tab via chrome context menu

Reply:
That was created as an eslint rule instead of defined in TS to allow this type of overriding specifically.

The eslint rule was created because it's not the general case that we want to allow but this is exactly the type of reason that is a good use case for overriding.

That said, if you can't get the `href` to be accurate then you're right that it's still not a good idea to use `href`.